internal/runners/push/push.go

```diff
@@ -91,7 +86,7 @@ func (r *Push) Run(params PushParams) error {
 	// Get the project remotely if it already exists
```

Comment:
As discussed on Slack I think we can just drop this altogether now.

Reply:
Looking at this more closely we need to keep the fetching of the remote project as we use that to determine if we're pushing to an existing project or creating a new project with `state init`. However, there is an error case that is no longer relevant and has been removed.